README.md

```diff
@@ -7,10 +7,27 @@ Python support for Sublime's LSP plugin provided through [microsoft/pyright](htt
 1. Install [LSP](https://packagecontrol.io/packages/LSP) and
    [LSP-pyright](https://packagecontrol.io/packages/LSP-pyright) via Package Control.
 1. Restart Sublime.
+1. Configure pyright for your virtualenv.
 
 ## Configuration
 
 There are some ways to configure the package and the language server.
 
 - From `Preferences > Package Settings > LSP > Servers > LSP-pyright`
 - From the command palette `Preferences: LSP-pyright Settings`
+
+Project specific settings can also be set for LSP-pyright (and all LSP plugins):
+
+- From the command palette `Project: Edit Project`
+
+Your project will also require a `pyrightconfig.json` file, usually in the project root.
+
+At a minimum, the file should define where your Python virtualenvs are located and the name of the one to use for your project:
+
+```json
+{
+    "venvPath": "/path/to/virtualenvs/",
+    "venv": "env"
+}
+```
```

Comment:
No, a project doesn't require a configuration file. It's all optional. Pyright will use global python environment if it's not configured explicitly. Only if you have specific needs for your project you need to configure virtual env.

Reply:
Would this okay? 
https://github.com/sublimelsp/LSP-pyright/pull/35/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L23

Reply:
> No, a project doesn't require a configuration file. It's all optional. Pyright will use global python environment if it's not configured explicitly. Only if you have specific needs for your project you need to configure virtual env.

Python devs have been using virtual envs pretty much exclusively for what, the last 15 years? Only if you have specific needs will you NOT need this configuration. That is what was so frustrating about getting this set up in the first place: the complete lack of mention of a config that is required for 99% of use cases. I’m not sure why the nitpicking.

Reply:
Check improved readme in https://github.com/sublimelsp/LSP-pyright/pull/35/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L23 and see if you have any comments.

The fact is that for simple scripts that rely on the global environment people don't need to configure any virtual envs. Also for developing ST packages, it's not needed, and instead, the `python.analysis.extraPaths` setting might need to be adjusted.

So yes, I will have issues with implying that it's **required** to configure virtual envs.

Reply:
Looks much better!